cargo-bolero/src/rmc.rs

```diff
@@ -0,0 +1,27 @@
+use crate::{exec, test, Selection};
+use anyhow::{anyhow, Result};
+use bit_set::BitSet;
+use core::cmp::Ordering;
+use std::{
+    fs,
+    io::{BufRead, BufReader, Result as IOResult, Write},
+    path::{Path, PathBuf},
+    process::Command,
+};
+
+pub(crate) fn test(selection: &Selection, test_args: &test::Args) -> Result<()> {
+    let _ = selection;
+    let _ = test_args;
+    let mut rmc_cmd = String::from("cargo rmc");
+    rmc_cmd.push_str(" --function ");
+    rmc_cmd.push_str(&selection.test());
+    rmc_cmd.push_str(" --cbmc-args --object-bits 16");
+    Command::new("sh")
+          .env("RUSTFLAGS", "--cfg=fuzzing_rmc --cfg=fuzzing")
+          .arg("-c")
+          .arg(rmc_cmd)
```

Comment:
Why are we creating a subshell here? Why not call `cargo` directly?

```suggestion
    let mut cmd = Command::new("cargo");
    cmd
        .arg("rmc")
        .arg("--function")
        .arg(selection.test())
        .arg("--cbmc-args")
        .arg("--object-bits")
        .arg("16")
```

Reply:
I had trouble getting it to work without a subshell (I think it was giving a "No such file or directory" error). I might have messed up something though, so I'll try your suggestion.

Reply:
Just a hunch but I'm assuming you were passing `cargo rmc` and not just `cargo`